### ISSUE TITLE
Keyboard shortcuts for desktop

### ISSUE THREAD

**Issue by yuletide** [unicef/projectconnect-app#13]
I saw this mentioned elsewhere but just dropping it here again as something that would be super useful, especially when mapping on desktop, in the zone mapping looking at imagery, having these would speed up mapping significantly, combined with an increased batch size (maybe selectable at the start?)

**Comment by yuletide** [comment]
And here's a quick console snippet for anyone still mapping to drop in (every 10 schools it clears, but worth it)

```
function handleKeys(e){ 
e.key === "y" && document.querySelectorAll('.btn-outline-primary')[0].click();
e.key === "n" && document.querySelectorAll('.btn-outline-primary')[1].click();
e.key === "u" && document.querySelectorAll('.btn-outline-primary')[2].click();
e.key === " " && document.querySelector('.next-section .btn-primary').click();
 }
document.addEventListener('keyup', handleKeys);
```

**Comment by yuletide** [comment]
Copying here for internal reference the improved version by @planemad

```
function handleKeys(e){ 
e.keyCode == '38' && document.querySelectorAll('.btn-outline-primary')[0].click();
e.keyCode == '40' && document.querySelectorAll('.btn-outline-primary')[1].click();
e.keyCode == '37' && document.querySelectorAll('.btn-outline-primary')[2].click();
e.keyCode == '39' && document.querySelector('.next-section .btn-primary').click();
 }
document.addEventListener('keydown', handleKeys);
```

right : next
up: yes
down: no
left: unsure

**Comment by lacabra** [comment]
This has been implemented in lacabra/proco-map-app#112
